web_tooltip_field/fields.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Komit - Cuong Nguyen Mtm
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.fields import Field
+
+
+def _description_help(self, env):
+    model_name = self.base_field.model_name
+    help_tooltip = env["ir.model.fields.help.tooltip"].search(
+        [
+            ("field_id.name", "=", self.name),
+            ("model", "=", model_name),
+            ("help", "not in", [False, ""]),
+        ],
+        limit=1,
+    )
+    if help_tooltip:
+        return help_tooltip.help
+    if self.help and env.lang:
+        field_help = env["ir.translation"].get_field_help(model_name)
+        return field_help.get(self.name) or self.help
+    return self.help
+
+
+Field._description_help = _description_help
```

Comment:
Could you make this monkey patch in a post load hooks as mentioned in OCA's conventions https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id11

Also I'm wondering if we could reuse the parent method something such as:

```suggestion
_former_field_description_help = Field._description_help


def _description_help(self, env):
    model_name = self.base_field.model_name
    help_tooltip = env["ir.model.fields.help.tooltip"].search(
        [
            ("field_id.name", "=", self.name),
            ("model", "=", model_name),
            ("help", "not in", [False, ""]),
        ],
        limit=1,
    )
    if help_tooltip:
        return help_tooltip.help
    return _former_field_description_help(self, env)


Field._description_help = _description_help
```

Reply:
Here you can find an example: https://github.com/OCA/server-tools/blob/16.0/module_change_auto_install/__manifest__.py#L15